src/classes/BGE_ConfigurationWizard_CTRL.cls

```diff
@@ -233,6 +233,20 @@ public with sharing class BGE_ConfigurationWizard_CTRL {
         return dfrs;
     }
 
+/*******************************************************************************************************
+    * @description Returns the corresponding Lightning field type from an Apex field type
+    */
+    public static String getLightningFieldType(String apexFieldType) {
+        if (apexFieldType == 'currency') {
+            return 'number';
+        } else if (apexFieldType == 'string') {
+            return 'text';
+        } else if (apexFieldType == 'boolean') {
+            return 'checkbox';
```

Comment:
Are there defined constants for the lightning field types to avoid hardcoding those strings as well?

Reply:
I haven't found any but I can create them.